src/components/overview/ProductInformation/ProductInformation.jsx

```diff
@@ -0,0 +1,30 @@
+import React, { useState, useEffect } from 'react';
+import AverageRating from './AverageRating.jsx';
+
+const ProductInformation = ({state, updateState}) => {
+
+  return (
+    <>
+      {state !== undefined && (
+        <div id='productinfo'>
+          <div id='rating'>
+            <div id='star'>
+              <AverageRating state={state} updateState={updateState}/>
+            </div>
+          </div>
+          <div id='category'>
+            [Category - {state.selectedProductCategory}]
+          </div>
+          <div id='productName'>
+            {state.selectedProductName}
+          </div>
+          <div id='price'>
+            {state.selectedProductDefaultPrice}
```

Comment:
Curious- have you begun to look at sale prices or will you do that in a later implementation? I have to include sale prices in my my module but I haven't seen any products with an actual sale value-- wondering if you have come across this yet.

Reply:
I believe the sale prices is inside of style API that I haven't implemented yet.
I think there's sale_price as a key. And if the sale_price's not available, the value would be null.

I'm planning to implement that when I setup Style Section today!

Reply:
Sounds good!